Dev Workflow article.

reynoldsalec opened the pull request:
Adapted this from an email I wrote...doesn't match our personas that well but wanted to have it as a reference to give to people like the folks I was talking with.

serundeputy commented on src/content/articles/agency-dev-workflow-2019.md:
```diff
@@ -0,0 +1,71 @@
+
+Your developers use SFTP because it's the fastest way to edit a site. Version control seems like a good idea, but it's been hard to find the right workflow that doesn't require shelling out money for Github accounts, managed hosting providers, and other accessories. You've heard of Docker and containers, but it's hard to think about moving off of hosting that's worked for years.
+
+As the creators of [Lando](https://www.devwithlando.io) and organizers of the Bay Area Drupal Camp DevOps Summit, we often hear from teams like yours. Often they're a bit afraid of being left behind, but also curious to see how other teams are handling the process of writing, testing, and deploying code.
```
~~https://www.devwithlando.io~~ --> https://docs.devwithlando.io

serundeputy commented on src/content/articles/agency-dev-workflow-2019.md:
```diff
@@ -0,0 +1,71 @@
+
+### 3. Local Development
+
+Use a Docker-based local development solution like [Lando](https://www.devwithlando.io). Your local development environment should support all the operating systems (Mac/Linux/Windows) and be able to adapt to all the application frameworks you create projects with (Drupal/WordPress/Laravel/NodeJS/etc.), but be simple enough that any new developer can get working within a couple minutes. Integrati ...
```
~~https://www.devwithlando.io~~ --> https://docs.devwithlando.io

serundeputy commented on src/content/articles/agency-dev-workflow-2019.md:
```diff
@@ -0,0 +1,71 @@
+
+## Your First Accessories
+
+Once you've adopted these components, most teams are curious to hear about this magical "automated testing" they've been hearing about. It's such a common topicI couldn't resist adding a few thoughts!
```
~~topicI~~ --> topic  I

dbungard commented:
My only thought would be to include both internal links (seo to services/ a11y to that page etc) as well as outgoing links to platform & pantheon and the other tools.

reynoldsalec commented:
Ok, finished writing some new section, added an image, and addressed @serundeputy's edits.

@dbungard let me know where else I could add some internal links (better yet you could [edit the file directly](https://github.com/thinktandem/thinktandem.io/edit/dev-workflow/src/content/articles/agency-dev-workflow-2019.md]).
